R/dsl-distributions.R

```diff
@@ -1,3 +1,29 @@
+##' Report information about supported distributions in the dsl.  This
+##' is primarily intended for use in packages which use
+##' [mcstate_dsl_parse_distribution], as this function reports
+##' information about which distributions and arguments would succeed
+##' there.
+##'
+##' @title Information about supported distributions
+##'
+##' @return A data.frame with columns
+##'
+##' * `name` the name of the distribution; each name begins with a
+##'   capital letter, and there are duplicate names where different
+##'   parametrisations are supported.
+##' * `args` the arguments of all parameters, *except* the random
+##'   variable itself which is given as the first argument to density
+##'   functions.
+##'
+##' We may expand the output here in future to include information on
```

Comment:
```suggestion
##' We may expand the output here in the future to include information on
```

Reply:
You quite often type "in future" without "the" - so perhaps this is a dialect issue!

Reply:
yeah, "in future" reads fine to me, but I can see that "in the future" is probably more Proper